README.md

```diff
@@ -1,2 +1,276 @@
-# example-application
-Example out-of-tree application that is also a module
+![Zephyr Project logo](https://github.com/zephyrproject-rtos/zephyr/blob/v1.14-branch/doc/images/Zephyr-Project.png)
+
+
+# Repository Overview
+
+The Zephyr Project is a scalable real-time operating system (RTOS) supporting
+multiple hardware architectures, optimized for resource constrained devices,
+and built with security in mind.
+
+The Zephyr build system supports integration of externally maintained
+projects called *modules*. This integration is configured through the
+settings in a *module.yml* file in a :file:`zephyr/` folder at the root
+of the repository.  See the [Zephyr Modules (External projects)][5]
+for a high-level overview of the build system module mechanism.
+
+
+## Scope
+
+The Zephyr eXample Application (ZXA) module is supplementary to the Zephyr
+repository.  This provides:
+
+1. A reference implementation of using the Zephyr module mechanism.
+
+1. A repository for holding example code and config files outside of the
+   Zephyr repository, including:
+
+   - `west` manifest (in the default location)
+   - projects/samples/tests
+   - applications/subsystems/drivers/interface declarations
+   - devicetree source files
+   - boards
+   - scripts
+
+1. A test case for CI to monitor Zephyr build system support for modules.
+
+
+   _WARNING:_ The Zephyr repository is intentionally
+   **independent of and unaware of this repository**.
+
+
+# Getting Started
+
+## Creating a New Module
+
+This module is designed to be a boilerplate for other modules simply by:
+
+1. cloning
+
+1. stripping out any unnecessary content (e.g. no drivers or subsystems)
+
+  - If you don't have any non-sample, non-test projects:
+
+    1. remove the `zephyr/projects/` subdirectory
+    1. In `zephyr/modules.yml` remove the reference to `zephyr/projects`
+
+
+## Creating a Free-Standing Manifest Repository
+
+This example module is also an appropriate template for creating a
+manifest-only repository because, like modules that also contain a
+manifest file, they may contain the following:
+
+- documentation describing the scope and purpose of the workspace
+  configured by the manifest
+
+- configuration settings for CI pipelines
+
+- scripts unique to the breadth of the workspace
+
+To strip this repository down to be a manifest-only repository:
+
+  1. Remove all files and subdirectories referenced by `zephyr/module.yml`
+  1. Remove the `zephyr/include` subdirectory
+  1. Remove `zephyr/module.yml`
+
+
+## Checking Out Code
+
+The Zephyr module ZXA may be cloned and added to the Zephyr build system
```

Comment:
Again, use repository and remove ZXA. Applies to the rest

Reply:
One of the barriers to a scalable, extensible ecosystem is a lack of namespace management. If you want an example that doesn't include namespace management, provide a second repository.